conflictlog/memcached.go

```diff
@@ -184,21 +184,21 @@ func (m *MemcachedConn) setMeta(conn *mcc.Client, key string, vbNo uint16, body
 	binary.BigEndian.PutUint64(req.Extras[16:24], cas)
 	binary.BigEndian.PutUint32(req.Extras[24:28], options)
 
-	//conn.logger.Debugf("bytes = %v", req.Bytes())
-
-	//rsp, err := m.mccConn.Set(vbNo, key, 0, 0, val, &mcc.ClientContext{CollId: colId})
 	rsp, err := conn.Send(req)
 	if rsp != nil {
 		if rsp.Opaque != opaque {
 			err = fmt.Errorf("opaque value mismatch expected=%d,got=%d", opaque, rsp.Opaque)
 			return
 		}
+
 		if rsp.Status == gomemcached.UNKNOWN_COLLECTION {
+			m.logger.Debugf("got UNKNOWN_COLLECTION id=%d, body=%s", m.id, string(rsp.Body))
 			err = ErrUnknownCollection
 			return
 		}
+
 		if rsp.Status == gomemcached.NOT_MY_VBUCKET {
-			m.logger.Infof("got NOT_MY_BUCKET bucketName=%s", m.bucketName)
+			m.logger.Debugf("got NOT_MY_BUCKET id=%d, bucketName=%s", m.id, m.bucketName)
```

Comment:
Vbucket

Reply:
If you are referring to the typo then it has been fixed.